src/_common/content/components/media-item/media-item.ts

```diff
@@ -101,10 +109,6 @@ export default class AppContentMediaItem extends Vue {
 		return this.computedHeight > 0 ? this.computedHeight + 'px' : 'auto';
 	}
 
-	get isHydrated() {
```

Comment:
Why should we remove this? I feel like expressing when something is loaded/hydrated through a prop makes more sense than checking the specific case of `!!this.mediaItem` everywhere.
It's also better to have a getter in case the check ever changes.

Reply:
The only reason I did this is because the formatter was showing warnings in the Vue template that `mediaItem` could be `null` when checking for `isHydrated`. I agree that it previously looked better, but `isHydrated` wasn't seeming to pass on that information down the tree in the template.

Reply:
I think we should keep it with the getter. The templates complaining is pretty annoying, but I feel like we need a different solution for it. We could always make a new getter that's like hydratedMediaItem that would return the mediaItem as non-null, and then using that in the template to remove the warnings, haha